src/requirements.txt

```diff
@@ -12,8 +12,8 @@ regex
 markdown
 django-haystack>=2.8.0,<2.9.0
 whoosh>=2.7.2
-pycrypto
-pyScss>=1.3.4,<2.0
+pycryptodome
+git+https://github.com/Kronuz/pyScss@73559d04 # newest PyScss version, not yet released
```

Comment:
Why upgrading pyScss?

Reply:
The version on pypi is incompatible with python 3.11 
https://github.com/Kronuz/pyScss/issues/428